buildkite/test-browser-e2e.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -ex
+HERE=$(dirname $0)
+. $HERE/common
+
+function cleardocker() {
+  docker logs hint
+  $HERE/../scripts/clear-docker.sh
+}
+trap cleardocker EXIT
+
+# Run all dependencies
+$HERE/run-dependencies-for-integration-tests.sh
+
+# Wait for HINT to become responsive
+sleep 10
+
+# Create an image based on the shared build env that compiles and tests the front-end ADR integration
+docker build --tag hint-test-browser-e2e \
+  --build-arg GIT_ID=$GIT_ID \
+  -f $HERE/test-browser-e2e.dockerfile \
+  .
+
+# Run the created image
+docker run --rm \
+  -e CI=$CI \
```

Comment:
What do you think about just setting the base URL here?

Thinking about the case that we want to debug the docker build, if we try and run this locally we will need to remember to set CI env or it won't work locally. If we just set the base URL here then it will avoid that issue?

A small point as it might not be a common way we run the tests but it will be useful if the tests are failing on CI but passing locally.

Reply:
`process.env.CI ` is used for other things, i.e setting the playwright workers for build kite and local ?

When debugging playwright docker container, it will display baseUrl in the console. If we have need for further things we will definitely extend it, for now this is fine.